Src/Skins/Immersive/Immersive.vcxproj

```diff
@@ -33,7 +33,87 @@
     <Text Include="SkinDescription.txt" />
   </ItemGroup>
   <ItemGroup>
-    <Image Include="*.png" />
```

Comment:
This is unwanted change, please, revert it.

> Because VS2022 warning ".vcxproj files and wildcards"

I've noticed the warning, but it seems that everything works without any issue.
I was thinking of defining `ReplaceWildcardsInProjectItems` property as the MS page recommends, but didn't get to it yet.

Reply:
> I was thinking of defining ReplaceWildcardsInProjectItems property as the MS page recommends

In fact that's not good idea either as with that property VS will populate all the existing files matching wildcard and it will get saved to disk when project is saved.

Better solution is to use `ReadOnlyProject` property.
As these _skin_ projects are basically created just once and then they are not supposed to be changed (from VS).

I will do that eventually.

Reply:
It seems to me that the absence of warnings is convenient, not distracting.
Among other things, I have already made changes that exclude warnings. ;-)

As far as I understand, need to create a resource header file resource.h and add 3/5 lines to the end of the files:
Immersive[7].vcxproj.filters
>--------------------<
   <ItemGroup>
     <ClInclude Include="resource.h">
       <Filter>Resource Files</Filter>
     </ClInclude>
   </ItemGroup>
>--------------------<
</Project>

Immersive[7].vcxproj
>--------------------<
   <ItemGroup>
     <ClInclude Include="resource.h" />
   </ItemGroup>
>--------------------<
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
</Project>

But I haven't figured out how...

Reply:
I've pushed the change to master already https://github.com/Open-Shell/Open-Shell-Menu/commit/d0ad10429f51536dde3f99c7f0397e7c22084d86